src/load_data.py

```diff
@@ -1,118 +1,60 @@
-import pandas as pd
+import boto3
+import logging
+import botocore
+import argparse
+import yaml
+import sys
+from os import path
+
+rel_path = path.dirname(path.dirname(path.abspath(__file__)))
+sys.path.append(rel_path)
+
+import config
+logger = logging.getLogger(__name__)
+
+
+def load_s3(args):
+    """
+    Copies raw dataset from public S3 bucket to user-defined bucket and folder
+    :param args: User arguments for public s3 path, public s3 folder, destination s3 bucket, destination s3 folder
+    :return: None
+    """
+    object_name = args.s3folder+"/EA_FIFA_19.csv"
+    s3 = boto3.resource('s3')
+    copy_source = {
+        'Bucket': args.public,
+        'Key': args.path
+    }
+    bucket = s3.Bucket(args.s3bucket)
+    try:
+        bucket.copy(copy_source, object_name)
+    except botocore.exceptions.NoCredentialsError as e:
+        logger.error(e)
+
+if __name__ == '__main__':
```

Comment:
Think Chloe mentioned, that since these functions will only be called through run.py, you don't need this "main" part here.